lib/Graph/Nodes.cpp

```diff
@@ -646,6 +646,10 @@ static bool verifyEmbeddingBag(NodeValue dest, NodeValue data,
   return isValid;
 }
 
+bool HardSigmoidNode::verify() const {
+  return checkSameType(getInput(), getResult(), this);
+}
+
```

Comment:
Don't need this anymore.

Reply:
done
4f1486f6cf04cbeca26a0ea40a631307ea0e0448